frontends/web/src/utils/qttransport.ts

```diff
@@ -37,36 +39,39 @@ function initTransport() {
       };
       check();
     } else if (runningInQtWebEngine()) {
-      const initWebChannel = function(channel) { // eslint-disable-line func-style
+      const initWebChannel = function(channel: any) {
         webChannel = channel;
-        webChannel.objects.backend.gotResponse.connect((queryID, response) => {
+        webChannel.objects.backend.gotResponse.connect((
+          queryID: number,
+          response: string,
+        ) => {
           queryPromises[queryID].resolve(JSON.parse(response));
           delete queryPromises[queryID];
         });
-        webChannel.objects.backend.pushNotify.connect(msg => {
+        webChannel.objects.backend.pushNotify.connect((msg: string) => {
           currentListeners.forEach(listener => listener(JSON.parse(msg)));
         });
         resolve(webChannel);
+
       };
-      // @ts-ignore
-      cache = new QWebChannel(qt.webChannelTransport, initWebChannel); // eslint-disable-line no-undef
+      cache = new QWebChannel((window.qt as any).webChannelTransport, initWebChannel);
```

Comment:
why not type `qt` as `any` in the first place then? or make it of type `{ webChannelTransport: any; }` instead?